practice_app/api/views.py

```diff
@@ -1,3 +1,51 @@
 from django.shortcuts import render
+from django.http import JsonResponse
+import requests
 
-# Create your views here.
+
+def doaj_get(request):
+    DOAJ_MAX_ROW = 10
```

Comment:
is this the maximum number allowed by the DOAJ api?

Reply:
I guess I can get as many rows as I want. I just wanted to put a limit.